bridge/deploy/10_transfer_setTokens_allowTokens.js

```diff
@@ -72,6 +85,18 @@ async function setTokensKovan(allowTokens, deployer) {
   ]).send({from: deployer});
 }
 
+async function setTokensBscTestnet(allowTokens, deployer) {
+  await allowTokens.methods.setMultipleTokens([
+    {token: '0x6ce8da28e2f864420840cf74474eff5fd80e65b8', typeId: '0'}, //BTC
+    {token: '0x8babbb98678facc7342735486c851abd7a0d17ca', typeId: '1'}, //ETH
+    {token: '0xae13d989dac2f0debff460ac112a837c89baa7cd', typeId: '2'}, //BNB
+    {token: '0x5d47b6e7edfc82e2ecd481b3db70d0f6600fdef8', typeId: '4'}, //USDC
+    {token: '0x337610d27c682e347c9cd60bd4b3b107c9d34ddd', typeId: '4'}, //USDT
+    {token: '0x110887fc420292dce51c08504cee377872d0db66', typeId: '4'}, //BUSD
+    {token: '0x13878644c0f2c9c5c8a85da43ebc3bb74bbc05a9', typeId: '4'}, //DAI
+  ]).send({from: deployer});
```

Comment:
we should add the side token for RIF, it should look like btRIF or something like that

Reply:
Do you mean like create an ERC20 at bsc testnet as btRIF ?

Reply:
i meant create a sideToken for tRIF on binanace using the Bridge and SideTokenFactory

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/269/commits/818e1adb30bf789f048ec0e6a47afe8c7eae0d81